packages/error-boundry-fallback/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@opensrp/error-boundry-fallback",
+  "version": "0.0.1",
+  "types": " dist/types",
+  "description": "Error boundry fallback component",
+  "main": "dist/index.js",
+  "repository": "https://github.com/opensrp/web",
+  "author": "Ona Engineering",
```

Comment:
I think the author is `OpenSRP Engineering`

Reply:
Not sure about the author, 
I followed info from other package's package.json file

Reply:
ok lets keep as its

Reply:
@p-netm @HaseemUlHaq I suggest we change it to `OpenSRP Engineering`